schema/deploy/function_get_swrs_operator_contact_data.sql

```diff
@@ -54,12 +54,16 @@ begin;
           _contact.email_address,
           _contact.telephone_number,
           _contact.fax_number,
-          cast('unit '  || ' ' || _contact_add.mailing_address_unit_number
-                   || ', ' || _contact_add.mailing_address_street_number
-                   || ' ' || _contact_add.mailing_address_street_name
-                   || ' ' || _contact_add.mailing_address_street_type
-                   || ' ' || _contact_add.mailing_address_street_direction as varchar(1000))
-                  ,
+          cast(
+            concat(
+              'unit ' || _contact_add.mailing_address_unit_number || ', ',
+              _contact_add.mailing_address_street_number || ' ',
+              _contact_add.mailing_address_street_number_suffix || ' ',
+              _contact_add.mailing_address_street_name || ' ',
+              _contact_add.mailing_address_street_type || ' ',
+              _contact_add.mailing_address_street_direction
+            )
+          as varchar(1000)),
```

Comment:
@hamzajaved This allows for fields to be `null`, while not printing "unit" if there is no unit number.
If one of the strings is null `||` outputs `null`, while `concat` ignores `null` parameters

Reply:
It saddens me that we have db testing set up in [cas-ggircs](https://github.com/bcgov/cas-ggircs/tree/develop/test) but not here. This is exactly the sort of comment that should be described by a unit test.